Makefile

```diff
@@ -3,9 +3,10 @@ PREFIX := /usr
 BINDIR=$(PREFIX)/bin
 LIBEXECDIR := $(PREFIX)/libexec
 LOCALSTATEDIR := /var
+CLEARCONTAINERS_SUFFIX := clear-containers
 
 SOURCES := $(shell find . 2>&1 | grep -E '.*\.(c|h|go)$$')
-PROXY_SOCKET := $(LOCALSTATEDIR)/run/clearcontainers/proxy.sock
+PROXY_SOCKET := $(LOCALSTATEDIR)/run/$(CLEARCONTAINERS_SUFFIX)/proxy.sock
```

Comment:
If you're fine with hardcoding this "clear-containers" everywhere, I am fine with it.
@jodh-intel any thought on this ?